lib/arduino/FastLedManager.cpp

```diff
@@ -1,6 +1,6 @@
 #include "FastLedManager.hpp"
 
-#include "../generic/DeviceDescription.hpp"
+#include "../device/DeviceDescription.hpp"
```

Comment:
You should use the `#include <DeviceDescription.hpp>` style for anything which is in `lib/`.